odoo/addons/base/views/res_device_views.xml

```diff
@@ -95,7 +82,7 @@
             <field name="name">User Devices</field>
             <field name="res_model">res.device</field>
             <field name="path">user-device</field>
-            <field name="view_id" ref="res_device_view_tree"/>
+            <field name="view_mode">list,kanban,form</field>
```

Comment:
why?

Reply:
User would not be able to switch to Kanban View without my patch.

Through:  `Settings/Technical/Security/ User Devices`  only `list` view is visible for `res.device` without my patch

Reply:
ok